### PR TITLE
qt: Add -custom-css-dir commmand line parameter

### PR DESCRIPTION
**Pull request by xdustinface**
This PR ist part of a series of +-25 PRs related to UI redesigns. Its ancestor is #3556, its successor is #3558. I did not screenshot every single PR and its changes, instead i made "walk through all screen" videos with the result of this PR series and also with the 0.15 UI. If there are any concrete screenshots wanted, just let me know. To build with the full set of changes you can build from the branch [xdustinface:pr-ui-redesign](https://github.com/xdustinface/dash/tree/pr-ui-redesign) which always contains all changes.

[ -> Walk through 0.15](https://youtu.be/cWQeHj5tWR0)
[ -> Walk through Redesign](https://youtu.be/0QeSyXo1aao)

I tried to give the commits enough text to make things obvious without a lot description for each PR. Also here, if you want more description for this specific PR, let me know.
### About this PR

This PR allows to load a custom directory as css directory. The css files in the provided directory will be used as stylesheets for the UI instead of the app internal ones.

### CODE REVIEW

**Comment by xdustinface** [comment]
ready for review

**Comment by xdustinface** [comment]
Those force pushed were only to figure how to reorder the commits in here properly because the order was messed. Now they appear in the correct order.

File: src/qt/dash.cpp [hunk 1]
```diff
@@ -740,6 +740,39 @@ int main(int argc, char *argv[])
...
+                    vecRequiredFiles.erase(itFile);
+                } else {
+                    strMissing += strFile + "\n";
+                }
+            }
+        }
+
+        if (vecRequiredFiles.size()) {
+            QMessageBox::critical(0, QObject::tr(PACKAGE_NAME),
+                                  QObject::tr("Error: %1 CSS file(s) missig in -custom-css-dir path.").arg(vecRequiredFiles.size()) + "\n\n" + strMissing);
```
[the remark this answers is not in the document]
Comment (xdustinface) [reply src/qt/dash.cpp]: Good catch 👍  squashed into and force pushed!

File: src/qt/dash.cpp [hunk 2]
```diff
@@ -740,6 +740,39 @@ int main(int argc, char *argv[])
...
+    // Validate/set custom css directory
+    if (gArgs.IsArgSet("-custom-css-dir")) {
+        fs::path customDir = fs::path(gArgs.GetArg("-custom-css-dir", ""));
+        QString strCustomDir = QString::fromStdString(customDir.string());
+        std::vector<QString> vecRequiredFiles = GUIUtil::listStyleSheets();
+        QString strMissing, strFile;
+
+        if (!fs::is_directory(customDir)) {
+            QMessageBox::critical(0, QObject::tr(PACKAGE_NAME),
+                                  QObject::tr("Error: Invalid -custom-css-dir path.") + "\n\n" + strCustomDir);
```
Comment (PastaPastaPasta) [inline src/qt/dash.cpp:752]: Why the double new line?

Comment (xdustinface) [reply src/qt/dash.cpp:752]: To get a bit space before the list of missing files. Thanks for asking haven't tested that for a while and it now realised that i messed with it over the time, will force push soon. 
Bildschirmfoto 2020-07-11 um 05 07 14

**Pull request merged**